src/main/java/ch/njol/skript/expressions/ExprSeed.java

```diff
@@ -0,0 +1,83 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
+ */
+package ch.njol.skript.expressions;
+
+import org.bukkit.World;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.PropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.registrations.Classes;
+import ch.njol.skript.util.Getter;
+import ch.njol.skript.util.Time;
+import ch.njol.skript.util.Timeperiod;
+import ch.njol.skript.util.Timespan;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+
+@Name("Seed")
+@Description("The seed of a world.")
+@Examples("broadcast \"Seed: %seed of player's world%\"")
+@Since("2.2-dev35")
+public class ExprSeed extends PropertyExpression<World, Number> {
+
+	static {
+		Skript.registerExpression(ExprSeed.class, Number.class, ExpressionType.PROPERTY, "[the] seed[s] [(in|of) %worlds%]", "%worlds%'[s] seed");
```

Comment:
%worlds%'[s] seed**[s]**

Not sure about what is ``seed in %wold%``

Reply:
https://minecraft.gamepedia.com/Seed_(level_generation)

Reply:
seed in

Reply:
Next time, don't be rhetorical when proposing changes in a pull request.

Reply:
I'm talking about the ``in`` word, is "seed in %world%" a correct sentence? Looks like you copied the Time expression that is why.